packages/inquirer/lib/ui/baseUI.js

```diff
@@ -65,10 +65,23 @@ class UI {
 
 function setupReadlineOptions(opt) {
   opt = opt || {};
+  // Inquirer 8.x:
+  // opt.skipTTYChecks = opt.skipTTYChecks === undefined ? opt.input !== undefined : opt.skipTTYChecks;
```

Comment:
personally I wouldn't bother documenting/commenting this, maybe the folks maintaining the package will just rather have `skipTTYChecks=true` as default

Reply:
I think this check is valid and maybe deserves a 8.x version together with 7.0.7.

Reply:
fair enough 👍